### DISCUSSION TITLE
sf control with Shelly

### DISCUSSION THREAD

**Discussion by welli2009** [reinhard-brandstaedter/solarflow-control#146] [Q&A]
Hello everyone, 
I'm Jacob and I go to school in Berlin. We are trying to disconnect our Solarflow from the cloud, which is working quite well so far. But I have a problem with the regulation by our Shelly. So I wanted to ask if anyone could provide me with their solarflow-control config for the Shelly, or the changes I would need to make in the python code.

Many thanks in advance.

**Comment by reinhard-brandstaedter** [comment]
Hi @welli2009,

which Shelly are you using as a smartreader input?

kr,
Reinhard

**Comment by welli2009** [comment]
Hi Reinhard,
   
  
   we‘ve got a Shelly Pro 3EM.
   
  
   Thanks,
   
  
   Jacob

**Comment by reinhard-brandstaedter** [comment]
Hi @welli2009 ,

take a look a t[this](https://github.com/reinhard-brandstaedter/solarflow-control/issues/138)
It contains an example for including a Pro EM3.
If your payload is similar, this should work.

Reinhard
